go/vt/vttablet/tabletmanager/tm_init.go

```diff
@@ -882,7 +882,7 @@ func (tm *TabletManager) initializeReplication(ctx context.Context, tabletType t
 		log.Warningf("primary tablet in the shard record does not have mysql hostname specified, possibly because that tablet has been shut down.")
 		return nil, nil
 	}
-	if err := tm.MysqlDaemon.SetReplicationSource(ctx, currentPrimary.Tablet.MysqlHostname, currentPrimary.Tablet.MysqlPort, false /* stopReplicationBefore */, true /* startReplicationAfter */); err != nil {
+	if err := tm.MysqlDaemon.SetReplicationSource(ctx, currentPrimary.Tablet.MysqlHostname, currentPrimary.Tablet.MysqlPort, true /* stopReplicationBefore */, true /* startReplicationAfter */); err != nil {
```

Comment:
Looks like this was introduced in https://github.com/vitessio/vitess/pull/10881 then (so 15.0.0 and later). We should note that in the description and in the Related Issues. Suggestion there:
```
  - Follow-up to: https://github.com/vitessio/vitess/pull/10881
```

That change was made to address: https://github.com/vitessio/vitess/issues/10880

Reply:
Okay 👍